manifest.json

```diff
@@ -3,23 +3,20 @@
   "applications": {
     "gecko": {
       "id": "copypatch@kiszka.org",
-      "strict_min_version": "78.4.0",
-      "strict_max_version": "102.*"
+      "strict_min_version": "102.0"
     }
   },
   "name": "Copy Patch",
   "description": "Copy email content to clipboard for application as patch",
   "author": "Jan Kiszka",
-  "version": "2.1.3",
+  "version": "2.1.4",
   "homepage_url": "http://git.kiszka.org/copypatch.git",
   "icons": {
     "32": "copypatch.png",
     "64": "copypatch64.png"
   },
   "background": {
-    "scripts": [
-      "background-script.js"
-    ]
+    "page": "background.html"
```

Comment:
What's the advantage of this change?

Reply:
You use ES6 modules.

Reply:
https://developer.thunderbird.net/add-ons/mailextensions#background-page

Reply:
Ok, that allows the import later on - understood. But it causes this warning: `This page is in Quirks Mode. Page layout may be impacted. For Standards Mode use “<!DOCTYPE html>”.`

Reply:
I did add that line to background.html in my PR and I do not see the error here. Are you 100% sure it is caused by copy patch and not by another html page loaded somewhere?